python/triton/compiler/compiler.py

```diff
@@ -211,7 +213,48 @@ def filter_traceback(e: BaseException):
         e.__traceback__ = frames[0]
 
 
+class CompileTimer:
+
+    def __init__(self) -> None:
+        self.start: float = time.time()
+        self.prologue_end: float = -1.0
```

Comment:
why -1.0 and not None?

Reply:
Oh yeah, good question-- fixing up shortly